src/functional/fencil_processors/codegens/gtfn/gtfn_ir.py

```diff
@@ -134,12 +134,17 @@ class StencilExecution(Node):
 }
 
 
+class TagDefinition(Node):
+    name: Sym
+    alias: Union[str, SymRef] = ""  # TODO eve has problem with Optional[str]
```

Comment:
What’s the problem here? Maybe eve does not like nested Unions (`Optional[str]` is `Union[str, None]`)? Maybe `Union[str, SymRef, None]` works?

Reply:
I also tried that, but didn't work. I'll open a PR where we can address this issue.